volunteer.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: flexible
+title: Volunteer
+---
+
+<div>
+
+    <!-- call for volunteers placeholder -->
+    <div class="alert alert-info text-center" role="alert">
+        <p>Call for Volunteers will be opened soon...</p>
+    </div>
+
+    <!-- <div class="alert alert-info text-center" role="alert">
+        <p>Thanks to all the Volunteers who are doing a great work to organize <b>PyConf Hyderabad 2017</b>.</p>
+        <p>Call for Volunteers is <b>Closed now</b>.</p>
+        <p>If you want to attend <b>PyConf Hyderabad 2017</b> you can buy tickets from <a
+                href="http://pyconf.hydpy.org/#tickets">this link</a>.</p>
+    </div> -->
+    <p>PyConf Hyderabad is a community event. Here each and every task is done by a volunteer. And here is the
+        opportunity for You to be a part of the community.</p>
+    <p>But before you register to volunteer, there are few things you should be aware of </p>
+    <ul>
+        <li>Volunteering takes time and effort, sometimes a lot of it. For online activities like handling social
+            networks or handling registrations etc take a lot of time. And sometimes rapid decisions have to be made. So
+            if you're volunteering, make sure you have the time. If you're unavailable at certain time period of the
+            day, let us know beforehand.</li>
+        <li>If you're volunteering for offline activities e.g. on spot registration or settings up stages or handling
+            video recording, anything that has to be done on the venue during the event, there is a good chance that you
+            won't be able to attend the sessions you just registered for. If someone needs you, you have to be there.
+        </li>
+        <li>Volunteering is not a free way into the event. At PyConf Hyderabad, everybody buys tickets. Even the event
+            organizers buy tickets. We volunteer because we love Python and Python Conferences, and want the event to be
+            awesome.</li>
+        <li>Volunteers are the most important part of the event. Without the volunteers, the event cannot happen. So
+            even if you are attending PyCon for the first time, we encourage you to volunteer with whatever you can. If
+            you don't want to miss any sessions, then volunteer for smaller tasks.</li>
+    </ul>
+    <p>Once your name is on the form, keep an eye on the mailing list. Threads will be created on specific tasks asking
+        for volunteers. If you think you can do the task and have enough time for it, raise your hand. That's it. We'll
+        invite you to our whatsapp group and you can help out in the tasks there.</p>
+    <p>Also, if you have any question, we'd really appreciate if you go through the mailing list archives first, and see
+        if it's already answered. If not, you can send a mail to the <a target="_blank"
+            href="https://mail.python.org/mm3/mailman3/lists/hydpy.python.org/" disabled>mailing list</a>.</p>
```

Comment:
@ananyo2012 let me know if we need to updated the mailing list link.

Reply:
Looks good for now. We can reword later.